### PR TITLE
Feat: product listing page added

### PR DESCRIPTION
**Pull request by Sen-442b**
Features

- responsive

- mobile first approach 

Limitations
-poor design

### CODE REVIEW

**Comment by Sen-442b** [comment]
image

File: screens/product-listing-page/source/style.css [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+    word-break: break-all;
+
+}
+
+
+
+
+
+
+.heart-icon{
```
Comment (siddhigate) [inline screens/product-listing-page/source/style.css]: You can consider using a code formatter like Prettier for consistent spacing and clean code.

File: screens/product-listing-page/source/style.css [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+    
+}
+
+
+
+/*
+
+ For Component Library
+
+.slider[type="range"]{
```
Comment (siddhigate) [inline screens/product-listing-page/source/style.css]: You can remove the commented code for cleaner code

**Review by siddhigate — APPROVED** [review]
Everything else looks good to me!

**Comment by preethasri** [comment]
looks good

File: screens/product-listing-page/source/style.css (hunk 1 above)
Comment (Sen-442b) [reply screens/product-listing-page/source/style.css]: Sure, thanks for the suggestion

**Review by preethasri — APPROVED** [review]

**Pull request merged**